src/symbol/sprite_atlas.js

```diff
@@ -61,6 +61,8 @@ class SpriteAtlas extends Evented {
 
         if (ArrayBuffer.isView(pixels)) {
             pixels = new Uint32Array(pixels.buffer);
+        } else if (pixels.data) {   //Support ImageData correctly
```

Comment:
Any particular reason to add this? We don't document `Map#addImage` as accepting `ImageData`.